src/pages/mining.md

```diff
@@ -47,14 +47,9 @@ After this runs, you'll probably see some installation logs, and at the end you
 }
 ```
 
-We need to get some testnet BTC to that address. Grab the `btcAddress` field, and call the BTC faucet:
-
-```bash
-# replace <btc_address> with `btcAddress` property from your keychain
-curl -XPOST "https://stacks-node-api.blockstack.org/extended/v1/faucets/btc?address=<btc_address>" | json_pp
```

Comment:
I looked at the file and the `/btc` endpoint is mentioned a couple of more times - should these be updated as well?

Reply:
@hstove ping